lib/phoenix/tracker/state.ex

```diff
@@ -68,7 +68,14 @@ defmodule Phoenix.Tracker.State do
   Returns the causal context for the set.
   """
   @spec clocks(t) :: {name, context}
-  def clocks(%State{replica: rep, context: ctx}), do: {rep, ctx}
+  def clocks(%State{replica: rep, context: ctx, replicas: replicas} = state) do
+    down_replicas =
+      replicas
```

Comment:
This can be simplified to 
```elixir
down_replicas = for {key, state} <- replicas, state == :down, do: key
```

(I worked with @indrekj together in investigating and fixing this)